packages/react-data-grid-react-window/src/components/DataGridBody/renderDataGridBody.tsx

```diff
@@ -0,0 +1,31 @@
+/** @jsxRuntime classic */
+/** @jsx createElement */
+
+import { createElement } from '@fluentui/react-jsx-runtime';
```

Comment:
Please declare a dependency on `@fluentui/react-jsx-runtime`

Reply:
What's the plan for this going forward - it's still an `alpha` release package. Do we expect to have this in react-components in the future?